src/NetAnalyzers/UnitTests/Microsoft.NetCore.Analyzers/Security/DataSetDataTableInWebSerializableObjectGraphTests.cs

```diff
@@ -31,7 +30,7 @@ public string MyWebMethod(DataTable dataTable)
     }
 }
 ",
-                GetCSharpResultAt(10, 31, "DataTable", "DataTable"));
+                GetCSharpResultAt(10, 41, "DataTable", "DataTable"));
```

Comment:
With the old technique, the first column of the location was the data-type, now we "correctly" report on the parameter.

Reply:
Tagging @dotpaul as this is a security rule.

Reply:
In this case, it's the type that's the issue, so I think I prefer the older technique's location.

Also, could you make all the security rule changes in the 2.9.x branch please?

Reply:
Sure for the 2.9.x and regarding the location I wasn't sure as it was originally reporting on the full parameter (type + var name), I'll change it.

Reply:
Thanks @Evangelink !